compiler/z/codegen/S390GenerateInstructions.cpp

```diff
@@ -3380,3 +3240,67 @@ TR::Instruction *generateZeroVector(TR::Node *node, TR::CodeGenerator *cg, TR::R
    {
    return generateVRIaInstruction(cg, TR::InstOpCode::VREPI, node, vecZeroReg, 0, 3);
    }
+
+#ifdef J9_PROJECT_SPECIFIC
+
+/**
+ * \brief This function returns true if the spcified instruction is designed to throw decimal overflow
```

Comment:
These types of statement should go under `\return` clause. Perhaps something like this:

```cpp
/**
 * \brief 
 *    Determines if an instruction can throw decimal overflow.
 * 
 * \param op 
 *    The instruction to check.
 * 
 * \returns
 *    true if \p op is designed to throw decimal overflow exception according to the PoP; false otherwise.
 *
 * \details 
 *    Linux on z runs a process with this exception disabled. As a result of this, the JIT signal handler
 *    does not receive a signal in case decimal overflow exception happens. This helper function decides 
 *    whether to rely on condition code in such cases.
*/
```